core_backend/app/contents/models.py

```diff
@@ -116,6 +113,53 @@ def __repr__(self) -> str:
         )
 
 
+class ContentForQueryDB(Base):
+    """
+    ORM for storing what content was returned for a given query.
+    Allows us to track how many times a given content was returned in a time period.
+    """
+
+    __tablename__ = "content_for_query"
+
+    content_for_query_id: Mapped[int] = mapped_column(
+        Integer, primary_key=True, nullable=False
+    )
+    user_id: Mapped[int] = mapped_column(
+        Integer, ForeignKey("user.user_id"), nullable=False
+    )
+    content_id: Mapped[int] = mapped_column(
+        Integer, ForeignKey("content.content_id"), nullable=False
+    )
+    query_id: Mapped[int] = mapped_column(
+        Integer, ForeignKey("query.query_id"), nullable=False
+    )
```

Comment:
let's swap these columns?

```suggestion
    query_id: Mapped[int] = mapped_column(
        Integer, ForeignKey("query.query_id"), nullable=False
    )
    content_id: Mapped[int] = mapped_column(
        Integer, ForeignKey("content.content_id"), nullable=False
    )
```